hibernate-core/src/main/java/org/hibernate/persister/entity/AbstractEntityPersister.java

```diff
@@ -4178,6 +4157,42 @@ protected void createLoaders() {
 		);
 	}
 
+	private UniqueEntityLoader getLoaderByLockMode(LockMode lockMode) {
+		if ( LockMode.NONE.equals( lockMode ) ) {
```

Comment:
LockMode is an enum. You can use == to compare values. Same applies to the line 4164.

Reply:
I prefer it like that. Old habit from my early Sonar days.